index.html

```diff
@@ -9,26 +9,183 @@
-    	<a class="connect">LET’S CONNECT</a>
-	<ul class="social-media">
-
-	<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt='' title=''></a></li>
-	<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
-	  
-	</ul>
-    </section>
+    <div class='mainsection'>
+	    <header class="header">
+	      <a class="mylogo">PRATAP PANABAKA</a>
+	      <nav class='nav'>
+		<img src='images/hamburg.svg' alt="Click for the Menu Options" title=''>
+	      </nav>
+	    </header>
+	    <section>
+	     	<p class="myheader">I’m PRATAP Glad to see you!</p>
+	    	<p class="content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+	    	<h3 class="connect">LET’S CONNECT</h3>
+		<ul class="social-media">
+
+		<li class="sm"><a href="https://twitter.com/PRATAP_TWT"><img src='images/twitter.svg'  width='20' height='20' alt="Twitter Profile" title=''></a></li>
+		<li class="sm"><a href="https://www.linkedin.com/in/pratap-kumar-panabaka-755489236/"><img src='images/linkedin.svg'  width='20' height='20' alt="LinkedIn Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/m.svg'  width='20' height='20' alt='' title=''></a></li>
+		<li class="sm"><a href="https://github.com/PRATAP-KUMAR"><img src='images/github.svg'  width='20' height='20' alt="GitHub Profile" title=''></a></li>
+		<li class="sm"><a href=""><img src='images/v.svg'  width='20' height='20' alt='' title=''></a></li>
+		  
+		</ul>
+	    </section>
+    </div>
+    <div class='projects-section'>
+    <div class="project-wrapper grid-box">
+    
+    	<div class="project">
+    		<img src="projects/project1.png" alt="project 1" title="Project 1" width="295" height="220" />
+    		<div class="project-heading">
+    			<h2 class="project-title">Project 1</h2>
+    			<div class="project-meta-data">
+    				<div class="project-client">Linux</div>
+    				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-tech">Linux User</div>
+				<img src='images/dot.png'  width='8' height='8' alt="Separator Dot" title=''>
+				<div class="project-year">2015</div>
+			</div>
+			<ul class="project-tags">
+
+				<li class="tags">html</li>
+				<li class="tags">css</li>
+				<li class="tags">javaScript</li>
+			
+			</ul>
```

Comment:
- [ ] Great job styling the technologies used correctly. However, kindly note that according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) that you chose, this section is supposed to come after the `<div class="project-description">`. Kindly fix this.

**This is how your design looks right now:**
![image](https://user-images.githubusercontent.com/86318284/176886315-0fc580fd-5c9d-40a4-9883-c0afa8b3c49b.png)

**This is the desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/176886447-bd98b38d-c729-403b-abc9-528ac1d6c90f.png)

**Kindly make sure you change this**